openapi.yml

```diff
@@ -186,6 +153,12 @@ paths:
           description: The IPA code of the organization the documents belong to
           schema:
             <<: *non-empty-string
+      requestBody:
+        description: The list of requests whose documents are to be signed and sent via email
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/RequestIdCollection"
```

Comment:
Because in the [onboarding flow](https://projects.invisionapp.com/share/DXSW5T29UNV#/screens/378155593) the first delegate is supposed to send two documents to the administration (), and I assumed both of them has to be sent in a single email. Of course if this is not a requirement, then the endpoint can be changed and the single request id can be included in the endpoint path. How should I proceed?

Reply:
That's right ! A thing: if the requests status changes according to the action success / failure, I'd return the whole requests payloads (instead of just ids).